labextension/package.json

```diff
@@ -1,31 +1,14 @@
 {
   "name": "@jupyterhub/jupyter-server-proxy",
   "version": "4.1.1-0.dev",
```

Comment:
If the labextension was published to `@jupyterhub/jupyter-server-proxy` on npmjs (e.g. if it exposed some extension point for the features in #395) then the README page would look pretty bad indeed. Recommend keeping these and/or generating them from `pyproject.toml` as the single source of truth.

npm and its raft of things has a different schema for valid versions, which would also need to be take into account.

Reply:
> npm and its raft of things has a different schema for valid versions, which would also need to be take into account.

This is valid semver, where `-0.dev` suffix will make it "earlier" than a `-beta.1` suffix. When publishing the helm charts dev releases we had to use valid semver versions and concluded this was a good dev release first part of a version suffix to avoid issues.

I'll capture the readme action point in a summary below and mark this as reslved for now.

Reply:
Hmmm wait so the README.md file is still around and it takes precidence over the description so things would look good still I think, see https://www.npmjs.com/package/@jupyterhub/jupyter-server-proxy.

I'll add misc things back and avoid the complexity of figuring anything out in detail.